spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
 require 'capybara/rails'
 require 'capybara/rspec'
 require 'capybara/poltergeist'
+require 'sucker_punch/testing/inline'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.